account_bank_statement_import_camt/README.rst

```diff
@@ -8,6 +8,19 @@ Module to import SEPA CAMT.053 Format bank statement files.
 
 Based on the Banking addons framework.
 
+Configuration
+=============
+
+The user can configure the way CAMT bank statements are imported:
+
+* Go to *Settings* -> *Configuration* -> *Accounting*
```

Comment:
Here the readme refers to the general settings but you actually allow the configuration from the journal instead.

In our solution for 11.0 (https://github.com/OCA/bank-statement-import/pull/174) we decided to allow the configuration from the general settings because we are convinced that the configuration per journal is not needed.
Did you get that kind of requirements from any customer?